source/write/delete.txt

```diff
@@ -0,0 +1,180 @@
+
+.. io-code-block::
+   :copyable: true
+
+   .. input:: /includes/write/delete.rb
+      :start-after: start-delete-one
+      :end-before: end-delete-one
+      :language: ruby
+      :dedent:
+   
+   .. output::
+      :language: console
+      :visible: false
+
+      Deleted 1 document(s)
+
+Delete Multiple Documents
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following example uses the ``delete_many`` method to remove all documents
+in which the value of the ``borough`` field is ``"Brooklyn"`` and the
+value of the ``name`` field is ``"Starbucks"``:
+
+.. io-code-block::
+   :copyable: true
+
+   .. input:: /includes/write/delete.rb
+      :start-after: start-delete-many
+      :end-before: end-delete-many
+      :language: ruby
+      :dedent:
+   
+   .. output::
+      :language: console
+      :visible: false
+
+      Deleted 3 document(s)
+
+Customize the Delete Operation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can pass a ``Hash`` object as a parameter to the ``delete_one`` and ``delete_many`` methods
+to set options to configure the delete operation. If you don't specify any
+options, the driver performs the delete operation with default settings.
+
+The following table describes the options that you can use to
+configure the delete operation:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Option
+     - Description
+
+   * - ``collation``
+     - | Specifies the kind of language collation to use when sorting
+         results. For more information, see :manual:`Collation </reference/collation/#std-label-collation>`
+         in the {+mdb-server+} manual.
+
+   * - ``session``
+     - | Specifies the session to use for the operation. To learn more about sessions, see
+         :manual:`Client Sessions and Causal Consistency Guarantees </core/read-isolation-consistency-recency/#std-label-sessions>`
+         in the {+mdb-server+} manual.
+
+   * - ``hint``
+     - | Specifies the index to use when matching documents.
+         For more information, see the :manual:`hint </reference/command/delete/#std-label-deletes-array-hint>`
+         option in the ``delete`` reference page of the {+mdb-server+} manual.
+
+   * - ``let``
+     - | Provides a map of parameter names and values to set top-level
+         variables for the operation. Values must be constant or closed
+         expressions that don't reference document fields. For more information,
+         see the :manual:`let
+         </reference/command/delete/#std-label-delete-let-syntax>` option in the ``delete``
+         reference page of the {+mdb-server+} manual.
+
+Modify Delete Example
+`````````````````````
+
+The following code specifies the ``hint`` option to
+instruct the delete operation to use the ``"name_index"`` index. Then, the example uses the
+``delete_many`` method to delete all documents in the ``restaurants``
+collection with a ``name`` field value that includes the string ``"Red"``.
```

Comment:
The code example here is using the `comment` option, not the `hint` option

Reply:
Whoops, made this change in my sandbox later in the PR without pasting the change into the proper directory.